### DISCUSSION TITLE
API Docs for non-public schema

### DISCUSSION THREAD

**Discussion by anshunjain** [supabase/supabase#19479] [Feature Requests]
It is apparent through a few threads in github and discord that non-public schemas, though supported technically through the REST API, do not have the dashboard API docs support (even after you expose schema and grant privileges). There are several people asking for it. This is fundamental to any resource organization. 

I would request the team to kindly consider this. Is there a principle or technical reason this cannot be done as yet?

**Comment by rykroon-momentum** [comment]
Yes, this would be a great feature to have.
